packages/atomic/src/pages/accessibility/commerce-full.html

```diff
@@ -8,98 +8,96 @@
     <script type="module" src="/build/atomic.esm.js"></script>
     <script nomodule src="/build/atomic.js"></script>
     <link rel="stylesheet" href="/themes/accessible.css" />
-    <script>
-      (async () => {
-        await customElements.whenDefined('atomic-search-interface');
-        const searchInterface = document.querySelector('atomic-search-interface');
+    <script type="module">
+      await customElements.whenDefined('atomic-search-interface');
+      const searchInterface = document.querySelector('atomic-search-interface');
 
-        await searchInterface.initialize({
-          accessToken: 'xxc23ce82a-3733-496e-b37e-9736168c4fd9',
-          organizationId: 'electronicscoveodemocomo0n2fu8v',
-          platformUrl: 'https://platform.cloud.coveo.com',
-          search: {
-            preprocessSearchResponseMiddleware: (response) => {
-              response.body.results.forEach((result) => {
-                if (!result.raw.ec_category || !result.raw.ec_category.length) {
-                  return;
-                }
-                const parentsElement = document.createElement('parents');
-                result.raw.ec_category
-                  .map((path) => path.split('|'))
-                  .forEach((path) => {
-                    const name = path.slice(-1)[0];
-                    const parentElement = document.createElement('parent');
-                    parentElement.setAttribute('name', name);
-                    parentElement.setAttribute(
-                      'uri',
-                      'https://electronics.coveodemo.com/plp/' +
-                        path
-                          .map((part) =>
-                            part
-                              .replace(/&\s/g, '')
-                              .replace(/[,:]?\s/g, '-')
-                              .toLowerCase()
-                          )
-                          .join('/')
-                    );
-                    parentsElement.appendChild(parentElement);
-                  });
-                result.raw.parents = '<?xml version="1.0" encoding="utf-16"?>' + parentsElement.outerHTML;
-              });
-              return response;
-            },
+      await searchInterface.initialize({
+        accessToken: 'xxc23ce82a-3733-496e-b37e-9736168c4fd9',
+        organizationId: 'electronicscoveodemocomo0n2fu8v',
+        platformUrl: 'https://platform.cloud.coveo.com',
```

Comment:
Should modify this sample to also use `organizationEndpoints` instead of `platformUrl` (deprecated)

Reply:
Moreover.... this whole file is dubious, filled with "hacks". IIRC this is an artefact leftover from the accessibility testing with Deque.

I would not be opposed to simply deleting it, since this is not how "commerce" should be done, in the future.